hw_14_sockets_client.py

```diff
@@ -0,0 +1,66 @@
+import datetime
+import socket
+from queue import LifoQueue
+from threading import Event, Thread
+from typing import Callable
+
+import keyboard
+import psutil
+
+
+class SensorMonitor:
+    """Performs threaded data acquisition from specified sensor."""
+
+    def __init__(self, source: Callable, interval: int):
+        self.source = source
+        self.interval = interval
+
+        self.data = LifoQueue()
```

Comment:
Could you explain why you are use LIFO queue here?

Reply:
In case sensor puts data in higher frequency than client reads it, data at the front of the queue will quickly become outdated. To prevent this, I've decided to provide data from the end of the queue.
Also, I should have implemented automatic cleanup of the queue.

Reply:
It's clear